client_plugin/vmdk_plugin/main_windows.go

```diff
@@ -20,36 +20,37 @@ import (
     log "github.com/Sirupsen/logrus"
 )
 
-var logger service.Logger
+// pluginServer is a wrapper for vDVS plugin so that the plugin can be run
+// as a service on Windows. Windows controls services by setting up callbacks
+// that is non-trivial. This wrapper utilizes a vendor library to handle this.
+type pluginServer struct{}
```

Comment:
We have another [`PluginServer`](https://github.com/vmware/docker-volume-vsphere/blob/a7641ebdba3fbc4bffb8d06f2c5c2b11fe0608fe/client_plugin/utils/plugin_server/plugin_server.go#L27) in our project, so I'm not sure if we should name thing something else.